src/cli/commands/wallet/import.rs

```diff
@@ -11,6 +12,9 @@ use crate::cli::commands::get_ipc_agent_url;
 use crate::cli::{CommandLineHandler, GlobalArguments};
 use crate::sdk::{IpcAgentClient, LotusJsonKeyType};
 
+const FVM_WALLET: u8 = 0;
```

Comment:
Why not an enum?

Reply:
I think we might not be able to pass enum to clap/cli, or at least not that easy. So use a number is just easier.